### PR TITLE
Reverting back the asctb schema

### PR DESCRIPTION
**Pull request by johardi**
tl;dr

### Slots that have been restored to their previous names.

Class: `AsctbRecord`
* as_records --> anatomical_structure_list
* ct_records --> cell_type_list
* bg_records --> gene_marker_list
* bp_records --> protein_marker_list
* bl_records --> lipid_marker_list
* bm_records --> metabolites_marker_list
* bf_records --> proteoforms_marker_list
* ref_records --> references

Class: `CellMarkerDescriptor`
* targeted_cell_type --> primary_cell_type
* cell_type_location --> primary_anatomical_structure
* characterizing_biomarkers --> biomarker_set

### Changes that do not impact production queries

* AnatomicalStructureInstance --> AnatomicalStructureColumnInstance
* CellTypeInstance --> CellTypeColumnInstance
* BiomarkerInstance --> BiomarkerColumnInstance
* Labels on the the column instances so they become less cryptic, e.g., 'Fallopian tube (fallopian-tube-R1-AS1)' -->  'Fallopian tube (Table fallopian-tube, Record 1, Column AS/1)'

### CODE REVIEW

**Comment by johardi** [comment]
Rename classes
* AnatomicalStructureColumnInstance --> AnatomicalStructureRecord
* CellTypeColumnInstance --> CellTypeRecord
* BiomarkerColumnInstance -- BiomarkerRecord
